Tab and chart

BourgeoisThibault opened the pull request.

rubenedery committed:
Add files via upload

[Add new documents to present tab and chart]

rubenedery committed:
Add files via upload

[Error about the last commit. This is the real commit with table and charts]

BourgeoisThibault commented:
@rubenedery  NON TU NE MERGE PAS !!!

SINON TU VAS NOUS AJOUTER 50 FICHIER...

BourgeoisThibault commented:
@rubenedery  un petit commit pour voir ton avancement?

@tarikou

rubenedery commented:
@BourgeoisThibault Je vais faire ca. J'ai retravaillé le code depuis la semaine dernière c'était un beau bordel. Je supprime ce que j'ai commit avant ou je garde les 2 versions?

rubenedery committed:
Add files via upload

Hello, I refactor my code since last week-end. 
All classes have changed.I renamed also few classes. 
I don't have finish but I hope soon !!
We will see you tomorrow for a debrief. Don't forget to fill excel file.

rubenedery commented:
CEST OK POUR MOI !! 
PS: @BourgeoisThibault  évite les pressions de la sorte :)

BourgeoisThibault commented:
Haha non mais cetait parce que je sais que tu as bosser mais je peux rien voir sans commit.

Faut surtout que tu fasse un gros ménage sinon tu va nous ajouter 50 fichiers

rubenedery commented:
T'inquiètes pas je fais surement ca demain !
Les maths me prennent un peu de temps!

BourgeoisThibault commented:
Obsolète
